pkgs/games/solicurses/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, ncurses
+}:
+
+stdenv.mkDerivation rec {
```

Remove unneeded `rec`

```suggestion
stdenv.mkDerivation {
```